NAMESPACE

```diff
@@ -7,6 +7,10 @@ export(cache_deactivate)
 export(cache_info)
 export(create_style_guide)
 export(default_style_guide_attributes)
+export(is_function_call)
```

Comment:
@lorenzwalthert I am not sure if it makes sense to export only `is_function_call()` without exporting its siblings, since they are all documented together and form a family:

- is_curly_expr(pd)
- is_cond_expr(pd)
- is_while_expr(pd)
- is_function_call(pd)
- is_function_dec(pd)
- is_comment(pd)

Should we export the others as well?

Reply:
Ok.